setup.py

```diff
@@ -39,8 +43,11 @@ def read(*rnames):
         'Natural Language :: English',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
```

Comment:
FWIW other zopefoundation projects are dropping Python 3.3 support.

Also, you're not running the tests on 3.3 (neither from Travis, nor from tox.ini), so I don't think it's right to claim it as a supported version in setup.py.